spec.html

```diff
@@ -4072,8 +4072,7 @@
       <emu-alg>
         1. If Type(_x_) is the same as Type(_y_), then
           1. Return the result of performing Strict Equality Comparison _x_ === _y_.
-        1. If _x_ is *null* and _y_ is *undefined*, return *true*.
-        1. If _x_ is *undefined* and _y_ is *null*, return *true*.
+        1. If _x_ is either *null*, *undefined*, or an undefined-like exotic object and _y_ is either *null*, *undefined*, or an undefined-like exotic object, return *true*.
```

Comment:
Would this benefit from a new abstract operation that emulates `== null`, and encapsulates all three of these comparisons?